.github/workflows/nightly.yml

```
@@ -349,7 +367,7 @@ jobs:
         with:
           token: ${{ secrets.ACTION_MONITORING_SLACK }}
   COVERAGE_DEBUG_NTS:
-    if: github.repository == 'php/php-src' || github.event_name == 'workflow_dispatch'
+    if: (github.repository == 'php/php-src') && (github.event_name != 'workflow_dispatch' || inputs.job == '*' || inputs.job == 'COVERAGE_DEBUG_NTS')
```

I think this is wrong. It wouldn't execute on `workflow_dispatch` in forks anymore.

```suggestion
    if: (github.repository == 'php/php-src') || (github.event_name == 'workflow_dispatch' && (inputs.job == '*' || inputs.job == 'COVERAGE_DEBUG_NTS'))
```